drag_and_drop_v2/public/css/drag_and_drop.css

```diff
@@ -83,8 +83,14 @@
     transform: translate(-50%, -50%); /* These blocks are to be centered on their absolute x,y position */
 }
 
+/* Focused option */
+.xblock--drag-and-drop .drag-container .item-bank .option:focus,
+.xblock--drag-and-drop .drag-container .item-bank .option:hover {
+    transform: scale(1.1);
```

Comment:
This is kinda cool, but...

In a case like this, I don't think it looks good when adjacent items meld together:
![screen shot 2015-12-18 at 11 37 46 am](https://cloud.githubusercontent.com/assets/945577/11905525/d7f4bf84-a57b-11e5-813c-8e5cf102a705.png)

And sometimes it doesn't actually stand out that much. e.g. in this example, it's not immediately obvious which block has focus:
![screen shot 2015-12-18 at 11 37 52 am](https://cloud.githubusercontent.com/assets/945577/11905536/e6637a2e-a57b-11e5-9873-f6447bb3e79f.png)

I think that a better visual effect would be simply a colored border, which is more standard. Try something like `    outline: 2px solid white; outline-offset: -4px;` and see what you think.

I also think that including this effect with `hover` may be a bit overkill/distracting, but a more subtle hover effect would be nice, and helps users know what they can interact with and what they cannot.